### PR TITLE
Add a simple fuzzing target for oak_functions loader

### PR DESCRIPTION
**Pull request by rbehjati**
Adds a simple fuzz target for `oak_functions/loader`. Allows re-enabling oss-fuzz for project oak.

Ref #2047

### CODE REVIEW

**Comment by rbehjati** [comment]
Also created https://github.com/google/oss-fuzz/pull/5719

File: oak_functions/loader/fuzz/fuzz_targets/create_response_and_apply_policy.rs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+        let response = Response {
+            status,
+            body,
+            length,
+        };
+
+        let policy = Policy {
+            constant_response_size_bytes: raw.int_in_range(50..=5000)?,
+            constant_processing_time: std::time::Duration::from_millis(10),
```
Comment (tiziano88) [inline oak_functions/loader/fuzz/fuzz_targets/create_response_and_apply_policy.rs:45]: Maybe also randomize this at some point?

**Review by tiziano88 — APPROVED** [review]
Awesome! How to actually run this? Or does it get run by runner already? Maybe add a short README somewhere?

**Comment by rbehjati** [comment]
> Awesome! How to actually run this? Or does it get run by runner already? Maybe add a short README somewhere?

I am trying to re-enable oss-fuzz (google/oss-fuzz#5719). That should be the main way to run fuzzing. 
I think it would be nice to have a command in the runner, and perhaps even include it in the CI with a limited number of runs (using the `-runs` option). 
But my current problem is that running `cargo fuzz coverage` fails in the docker. I think there is an LLVM problem. I am not sure how to upgrade to a newer version of LLVM. I tried with a newer debian image, but it did not fix the problem. But perhaps for the runner command we wouldn't need to run coverage. 

I am also looking into ways to make fuzzing faster. For example, adding a seed or an initial corpus could help.

It is a good idea to have README. I'll add one to the `docs` folder in a separate PR.

**Pull request merged**